src/main/java/appeng/core/Registration.java

```diff
@@ -556,7 +460,7 @@ void postInit( final FMLPostInitializationEvent event )
 		if( AEConfig.instance().isFeatureEnabled( AEFeature.VILLAGER_TRADING ) )
 		{
 			// TODO: VILLAGER TRADING
-			// VillagerRegistry.instance().getRegisteredVillagers()..registerVillageTradeHandler( 3, new AETrading() );
+			// VillagerRegistry.instance().getRegisteredVillagers().registerVillageTradeHandler( 3, new AETrading() );
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)